src/Parser.js

```diff
@@ -26,14 +26,35 @@ export default class Parser {
     }
 
     _parseAst(code) {
-        let tokens = [];
+        // let tokens = [];
```

Comment:
remove this and https://github.com/cst/cst/pull/64/files#diff-a0c375bdbb580536aa70722d5d376137R56 later